execution/executor/sortition.go

```diff
@@ -22,6 +22,9 @@ func (e *SortitionExecutor) Execute(trx *tx.Tx, sb sandbox.Sandbox) error {
 	if val == nil {
 		return errors.Errorf(errors.ErrInvalidTx, "Unable to retrieve validator")
 	}
```

Comment:
In some blockchain validators can rebond, but this is not safe. Imagine a hacker has access to the validator key, then it can rebond and send malicious votes to the network.

Reply:
[the lines below ](https://github.com/zarbchain/zarb-go/pull/123/files/6eac6392c331973bd133346ed78df485e34e0537#diff-4052820177a99798ca41230c6ab7c97f271a525379b0a2d831258c0c05a822f7R25-R27) check if power is 0 i wanted to have distinction of stack and power for this reason actually the power when ever `UnbondingHeight ` is set will return 0 indicating the fact validator has unbounded

```go
func (val Validator) Power() int64 {
	// Only bootstrap validators at genesis block has no stake
	if val.data.Stake == 0 {
		return 1
	} else if val.data.UnbondingHeight > 0 { //if the validator requested to unbound ignore stake
		return 0
	}
	return val.data.Stake
}
```

Reply:
Yes. I got it. This is nice. By this solution, you actually set validator power to zero. It looks its car engine stopped. Keep it. 

Only add this logic to Bond execution.